libqtile/hook.py

```diff
@@ -954,6 +936,11 @@ def _fire_async_event(co):
 
 def fire(event, *args, **kwargs):
     if event not in subscribe.hooks:
+        # Suppress error for custom events. Event is only in subscribe.hooks
+        # if a custom hook has been subscribed but it should be possible to fire custom
+        # hooks even if there are no subscriptions.
```

Comment:
Why should it be possible to fire a custom event if there are no subscriptions? What's the point?

Reply:
sound like another topic to me, i.e. how we handle third party hooks; I feel like they shouldn't enter in the "user hooks" category

Reply:
ok. I'll remove for now but it is something that I'd think about adding (unless I just do some code injection with qtile-extras...)